detection-rules/attachment_pdf_file_with_embedded_content.yml

```diff
@@ -36,7 +36,6 @@ source: |
         "rar_file",
         "tar_file",
         "xar_file",
-        "zip_file",
```

Comment:
@ajpc500 any issues with removing this? seeing some FPs.

Reply:
go for it!